src/index.css

```diff
@@ -10,4 +10,4 @@ body {
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
-}
```

Comment:
It doesn't look like you made any significant changes to this file but I don't  understand the purpose of this file. I suggest deleting it. If this styling is needed we should add it to one of the files in the Styles folder.

Reply:
@matthewcotton - i've pushed the carousel component up with a css file for the component